drivers/cache/Kconfig.stm32

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2025 Henrik Lindblom <henrik.lindblom@vaisala.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config CACHE_STM32
+	bool "STM32 cache driver"
+	select CACHE_HAS_DRIVER
+	depends on CACHE_MANAGEMENT
+	help
+	  Enable support for the STM32 ICACHE / DCACHE peripheral present in some Cortex-M33 chips.
```

Let's avoid this becoming stale with possible new products.
```suggestion
	  Enable support for the STM32 ICACHE / DCACHE peripheral present in some chips.
```